### ISSUE TITLE
Test the accessibility and performance of the application

### ISSUE THREAD

**Issue by elielijah321** [elielijah321/ASE-Sum-2-Square-Root-Checker#9]
As a developer, I want to test the accessibility and performance of the application so that I can ensure that the application is accessible to everyone and performs well under different conditions.

**Acceptance Criteria:**
- The accessibility and performance testing should be carried out using appropriate tools and techniques
- The accessibility and performance testing should cover all the key features of the application
- The accessibility and performance testing results should be used to improve the application's accessibility and performance

**Comment by elielijah321** [comment]
![Image](https://user-images.githubusercontent.com/45821029/232074693-9de4244a-3f5b-400c-b11b-d5e28f038528.png)
